psycopg3/psycopg3/connection.py

```diff
@@ -349,7 +349,7 @@ async def _start_query(self) -> None:
         if self._autocommit:
             return
 
-        if self.pgconn.transaction_status == TransactionStatus.INTRANS:
+        if self.pgconn.transaction_status == TransactionStatus.IDLE:
```

Comment:
Maybe != IDLE?

Reply:
Derp! Maybe I need to get some sleep.

Also, I should look at making async work on Windows so I can actually run these tests locally :-/